sample/src/main/kotlin/me/dmdev/rxpm/sample/main/util/PhoneUtil.kt

```diff
@@ -39,6 +44,31 @@ class PhoneUtil {
         return formattedPhone.replace(code, "").trim()
     }
 
+    fun formatPhoneNumber(phoneNumberString: String): String {
+
+        val phoneNumber = onlyPhone(phoneNumberString).take(MAX_PHONE_LENGTH)
+        var formattedPhone: String = phoneNumber
```

Comment:
Maybe change to something like this?

`val formattedPhone = phoneNumber.toCharArray().map { asYouTypeFormatter.inputDigit(it) }.last()`

Reply:
Thank you, I will refactor the sample in the following PRs